app/assets/javascripts/update-organization.coffee

```diff
@@ -0,0 +1,2 @@
+$(".update_organization").on 'click', ->
+  $('form.edit_organization').submit();
```

Comment:
Can you add a newline here? I need to add a JS linter to catch this kind of stuff.